stores/expenses/README.md

```diff
@@ -50,21 +50,26 @@ For more on modeling, check out:
 
 ### Model
 ```python
+model
+  # We are using the 1.1 schema with type restrictions
+  schema 1.1
+
```

Comment:
```suggestion
type user
```

Reply:
There is no user type in the expenses store. We have type employee instead

Reply:
In this case, how come line 61, line 69 etc. refers to user?

Reply:
Those are typos - fixed. Thanks!